modules/_labs/badge/css/index.scss

```diff
@@ -0,0 +1,3 @@
+@import '@workday/canvas-kit-css-core/index.scss';
```

Comment:
Q: Should I be more specific with which assets I import here?

Reply:
Yeah if all you're using is variables, I would narrow this scope